tests/robottelo/test_func_locker.py

```diff
@@ -0,0 +1,321 @@
+# coding: utf-8
```

Comment:
@abalakh @ldjebran they are synonyms

`# -*- encoding: utf-8 -*-` is the `airplane pattern, because of `-*-` this is used only by `emacs` editor to set python mode.

Python interpreter has a regex matching existence of `coding` and `:` and `<encoding code>` so both pattern will work.

Also you could use

`# myawesomecoding: utf-8` and this would also work :)